taker/src/main.rs

```diff
@@ -374,3 +377,33 @@ async fn resolve_maker_addresses(maker_addr: &str) -> Result<Vec<SocketAddr>> {
     );
     Ok(possible_addresses)
 }
+
+/// Attach this fairing to enable logging Rocket launch
+pub fn ui_browser_launch(
+    username: rocket_basicauth::Username,
+    password: rocket_basicauth::Password,
+) -> impl Fairing {
+    AdHoc::on_liftoff("ui browser launch", move |rocket| {
+        Box::pin(async move {
+            let http_endpoint = format!(
+                "http://{}:{}@{}:{}",
+                username,
+                password,
```

Comment:
You can get this from the `Rocket` instance: https://docs.rs/rocket/0.5.0-rc.1/rocket/struct.Rocket.html#method.state